src/modules/FFmpeg/FFDec.cpp

```diff
@@ -81,6 +81,7 @@ AVCodec *FFDec::init(StreamInfo &streamInfo)
 }
 bool FFDec::openCodec(AVCodec *codec)
 {
+    codec_ctx->flags |= AV_CODEC_FLAG_LOW_DELAY;
```

Comment:
I have to check it... Maybe apply only for live streams? Are there any side effects (bugs, higher CPU/GPU usage, poor multithreading, etc)? What about local files?

Reply:
sorry I could not really tell if that will cause any trouble. I modified a few more files to introduce `bool is_local` in StreamInfo, so that FFDec can recognise and enable LOW_DELAY flag for only live streams, is that what you prefer?